packages/fluentui/docs/src/prototypes/table/ResponsiveTable.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { TableCell, TableProps, Table } from '@fluentui/react-northstar';
+import calculateBreakpoints, { Column } from '../../utils/calculateBreakpoints';
+import * as _ from 'lodash';
+
+interface ResponsiveTableProps extends TableProps {
+  config: Column[];
+  children: React.ReactElement<typeof Table>;
+  breakpoints?: number[];
+  id?: string;
+}
+
+const ResponsiveTableContainer: React.FC<ResponsiveTableProps> = props => {
+  const { config, children: ChildrenTable } = props;
+
+  const responsiveTableID = React.useMemo(() => props.id || _.uniqueId(`${Table.className}__responsive-wrapper`), [
+    props.id,
+  ]);
+
+  const [breakpoints, setBreakpoints] = React.useState<{ [key: string]: Column[] }>();
+
+  // Generate the stylesheet once and for all
+  React.useLayoutEffect(() => {
+    setBreakpoints(calculateBreakpoints(config, props.breakpoints || [1200, 960, 600, 280]));
+  }, []);
+
+  return (
+    <div id={responsiveTableID}>
+      <style>
+        {breakpoints &&
+          Object.entries(breakpoints)
```

Comment:
Is not supported in IE11: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

We can use `_.map()` from Lodash here, in this case w will not check for `undefined`

Reply:
`_.toPairs(object)` would make more sense?

Reply:
Done with `_.toPairs(object)`